packages/gator-permissions-snap/snap.manifest.ts

```diff
@@ -4,9 +4,10 @@
 import type { SnapManifest } from '@metamask/7715-permissions-shared/types';
 
 const kernelSnapId = process.env.KERNEL_SNAP_ID;
-
+const messageSnapId = process.env.MESSAGE_SIGNING_SNAP_ID;
+const snapEnv = process.env.SNAP_ENV;
 const manifest: SnapManifest = {
-  version: '0.2.1',
+  version: '0.3.0',
```

Comment:
Not strictly related to this PR, but given we are in here:

It would be nice to read the version from package.json, rather than having to manually update this every release.